airbyte-integrations/connectors/source-chargebee/source_chargebee/schemas/addon.json

```diff
@@ -174,6 +174,20 @@
           }
         }
       }
+    },
+    "custom_fields": {
+      "type": ["null", "array"],
+      "items" : {
+        "type" : ["null", "object"],
+        "properties": {
+          "name" : {
+            "type" : ["null", "string"]
+          },
+          "value": {
+            "type": "string"
```

Comment:
nit: "type" : ["null", "string"]

Reply:
Please, make sure you've checked all the schemas accordingly

Reply:
thanks, done